systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthAbstractST.java

```diff
@@ -72,73 +67,31 @@ void setup() throws Exception {
-            .editSpec()
-                .editKafka()
-                    .editListeners()
-                        .withNewTls()
-                            .withNewKafkaListenerAuthenticationOAuth()
-                                .withValidIssuerUri(keycloakInstance.getValidIssuerUri())
-                                .withJwksExpirySeconds(keycloakInstance.getJwksExpireSeconds())
-                                .withJwksRefreshSeconds(keycloakInstance.getJwksRefreshSeconds())
-                                .withJwksEndpointUri(keycloakInstance.getJwksEndpointUri())
-                                .withUserNameClaim(keycloakInstance.getUserNameClaim())
-                                .withTlsTrustedCertificates(
-                                    new CertSecretSourceBuilder()
-                                        .withSecretName(SECRET_OF_KEYCLOAK)
-                                        .withCertificate(CERTIFICATE_OF_KEYCLOAK)
-                                        .build())
-                                .withDisableTlsHostnameVerification(true)
-                            .endKafkaListenerAuthenticationOAuth()
-                        .endTls()
-                        .withNewKafkaListenerExternalNodePort()
-                            .withNewKafkaListenerAuthenticationOAuth()
-                                .withValidIssuerUri(keycloakInstance.getValidIssuerUri())
-                                .withJwksExpirySeconds(keycloakInstance.getJwksExpireSeconds())
-                                .withJwksRefreshSeconds(keycloakInstance.getJwksRefreshSeconds())
-                                .withJwksEndpointUri(keycloakInstance.getJwksEndpointUri())
-                                .withUserNameClaim(keycloakInstance.getUserNameClaim())
-                                .withTlsTrustedCertificates(
-                                    new CertSecretSourceBuilder()
-                                        .withSecretName(SECRET_OF_KEYCLOAK)
-                                        .withCertificate(CERTIFICATE_OF_KEYCLOAK)
-                                        .build())
-                                .withDisableTlsHostnameVerification(true)
-                            .endKafkaListenerAuthenticationOAuth()
-                        .endKafkaListenerExternalNodePort()
-                    .endListeners()
-                .endKafka()
-            .endSpec()
-            .done();
-
-        KafkaTopicResource.topic(CLUSTER_NAME, TOPIC_NAME).done();
-
         createSecretsForDeployments();
+    }
+
+    @AfterEach
+    void tearDown() {
 
-        KafkaUserResource.tlsUser(CLUSTER_NAME, OAUTH_CLIENT_NAME).done();
+        for (Job job : kubeClient().getClient().batch().jobs().inNamespace(NAMESPACE).list().getItems()) {
+            LOGGER.info("Deleting {} job", job.getMetadata().getName());
+            kubeClient().getClient().batch().jobs().inNamespace(NAMESPACE).delete(job);
+        }
```

Comment:
Why not to use the `JobUtils.deleteJob()`?

And question -> don't we want to add something like `JobsClient()` to not always do the `kubeClient().getClient().batch().jobs()`? This is just a question/suggestion.

Reply:
Leftover, feel free to open issue to add this job implementation to kubeclient.

Reply:
Perfect, thanks